db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_tenants.rb

```diff
@@ -3,7 +3,7 @@
 #
 tenant_list = {nil => "default"}
 service_template = $evm.root.attributes["service_template"]
-if service_template.orchestration_manager
+if service_template.respond_to?(:orchestration_manager)
```

Comment:
Is an orchestration_manager an association?  If so, I believe `service_template.orchestration_manager` be nil even though `service_template` responds_to? the message.